src/modules/moon-pay/ModuleMoonPay.vue

```diff
@@ -169,18 +165,23 @@ export default {
       if (newVal) {
         this.orderHandler = new handler();
       }
+      this.selectedCurrency = {};
       this.selectedCurrency = this.defaltCurrency;
```

Comment:
typo in `this.defaltCurrency`. Other places too

Reply:
Thats how the property was named but I can change the prop name